.github/PULL_REQUEST_TEMPLATE/main-template.md

```diff
@@ -0,0 +1,6 @@
+Tag name (The PR title should also include the tag name):
+Originator(s):
+
+List all `develop` PR numbers included in this PR and the title of each:
+
+List all test failures:
```

Comment:
I might make the same change here that I recommended for the `development` PR template:

```suggestion
List all automated tests that failed, as well as an explanation for why they weren't fixed:
```

Reply:
changed!